mobile-app/app/screens/AppNavigator/screens/Portfolio/screens/SendConfirmationScreenV2.tsx

```diff
@@ -0,0 +1,305 @@
+    await send({
+      address: destination,
+      token,
+      amount,
+      networkName: network.networkName
+    }, dispatch, () => {
+      onTransactionBroadcast(isOnPage, navigation.dispatch)
+    }, logger)
+    setIsSubmitting(false)
+  }
+
+  function onCancel (): void {
+    if (!isSubmitting) {
+      navigation.navigate({
+        name: 'Send',
+        params: {
+          token
+        },
+        merge: true
+      })
+    }
+  }
+
+  return (
+    <ThemedScrollViewV2 style={tailwind('pb-4')}>
+      <ThemedViewV2 style={tailwind('flex-col px-5 py-8')}>
+        <SummaryTitleV2
+          amount={amount}
+          title={translate('screens/SendConfirmationScreen', 'You are sending')}
+          testID='text_send_amount'
+          iconA={token.displaySymbol}
+          fromAddress={address}
+          fromAddressLabel={addressLabel}
+          toAddress={destination}
+        />
+
+        {conversion?.isConversionRequired === true &&
+          <ThemedView
+            style={tailwind('border-t-0.5 pt-5 mt-8 mb-2')}
+            light={tailwind('bg-transparent border-mono-light-v2-300')}
+            dark={tailwind('bg-transparent border-mono-dark-v2-300')}
+          >
+            <NumberRowV2
+              containerStyle={{
+                style: tailwind('flex-row items-start w-full bg-transparent '),
+                light: tailwind('bg-transparent'),
+                dark: tailwind('bg-transparent')
+              }}
+              lhs={{
+                value: translate('screens/SendConfirmationScreen', 'Amount to convert'),
+                testID: 'amount_to_convert',
+                themedProps: {
+                  light: tailwind('text-mono-light-v2-500'),
+                  dark: tailwind('text-mono-dark-v2-500')
+                }
+              }}
+              rhs={{
+                value: conversion.conversionAmount.toFixed(8),
+                suffix: 'DFI',
+                testID: 'amount_to_convert_value',
+                themedProps: {
+                  light: tailwind('text-mono-light-v2-900'),
+                  dark: tailwind('text-mono-dark-v2-900')
+                }
+              }}
+            />
+            <View style={tailwind('flex flex-row text-right items-center justify-end')}>
+              <ThemedTextV2
+                style={tailwind('mr-1.5 text-sm font-normal-v2')}
+                light={tailwind('text-mono-light-v2-500')}
+                dark={tailwind('text-mono-dark-v2-500')}
+                testID='conversion_status'
+              >
+                {
+                  translate('screens/ConvertConfirmScreen',
+                    conversion?.isConversionRequired && conversion?.isConverted !== true
+                      ? 'Converting'
+                      : 'Converted')
+                }
+              </ThemedTextV2>
+              {conversion?.isConversionRequired && conversion?.isConverted !== true && <ThemedActivityIndicatorV2 />}
```

Comment:
We can compile the two conditions here. So it only displays when conversion is required

Reply:
have different condition, Updated, removed `conversion?.isConversionRequired` since it's unnecessary